starknet/src/space/space.cairo

```diff
@@ -5,28 +5,71 @@ use sx::types::{
 
 #[starknet::interface]
 trait ISpace<TContractState> {
-    // State 
+    // -- View functions --
+
+    /// Returns the owner of the contract.
     fn owner(self: @TContractState) -> ContractAddress;
-    fn max_voting_duration(self: @TContractState) -> u32;
+
+    /// The voting delay to wait for between a proposal is created and the start of the voting period.
+    fn voting_delay(self: @TContractState) -> u32;
+
+    /// Returns the minimum voting duration. Once this duration has elapsed,
+    /// a proposal can be finalized. Most DAOs should set `min_voting_duration` and
```

Comment:
technically a proposal can be executed before min voting duration has elapsed as we check that in the execution strategy. think just a simple comment like in sx-evm is fine: https://github.com/snapshot-labs/sx-evm/blob/df37a70673067300a3a6c0541740117a86d0bc59/src/interfaces/space/ISpaceState.sol#L13C6-L13C6

Reply:
6f348d79f7d366db2741e52a881e17708515063c